scripts/openshiftci-presubmit-all-tests.sh

```diff
@@ -27,7 +27,7 @@ chmod 640 $TMP_DIR/kubeconfig
 export KUBECONFIG=$TMP_DIR/kubeconfig
 
 # Login as developer
-oc login -u developer -p password@123
+oc login -u developer -p password@123 --insecure-skip-tls-verify
```

Comment:
I can see other occurrences of `odo login -u developer` in some of our scripts:
- [`scripts/openshiftci-Nightly-SBO-tests.sh`](https://github.com/redhat-developer/odo/blob/main/scripts/openshiftci-Nightly-SBO-tests.sh#L25)
- [`scripts/openshiftci-periodic-tests.sh`](https://github.com/redhat-developer/odo/blob/main/scripts/openshiftci-periodic-tests.sh#L24).

Do we need to update those too?

Reply:
We can add this to other script as well, but it's mostly required for the interop env.
But as it does no harm, I will add it.